tests/unit_tests/test_element.py

```diff
@@ -0,0 +1,75 @@
+import openmc.element
+import pytest as pt
+
+from openmc.data import NATURAL_ABUNDANCE, atomic_mass
+
+# Relative tolerance for float comparison
+TOL = 1e-9
+
+
+def test_expand_no_ernichment():
+    """ Expand Li in natural compositions"""
```

Comment:
There is a typo in the test name: should be corrected to test_expand_no_e**nr**ichment